hamocc/mo_ocprod.F90

```diff
@@ -667,6 +706,11 @@ subroutine ocprod(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,omask,ptho,pi_ph,psao,pp
               ocetra(i,j,k,izoo14) = ocetra(i,j,k,izoo14)-sterzo14
             endif
 
+            if (use_river2omip) then
+              tdoclc_deg = deg_tdoclc*ocetra(i,j,k,itdoc_lc) ! tDOC degradation independent from O2 (cf. R2OMIP)
+              tdochc_deg = deg_tdochc*ocetra(i,j,k,itdoc_hc)
+            endif
```

Comment:
@TimotheeBrgs , since you apply the changes inside the O2-threshold if statement (which I agree on), I would suggest to put these statements into the first appearance of your new `use_river2omip`  below (l. 765) (easier to read and isn't calculated in case  that it is later not applied).